### PR TITLE
Prevent panic when no integration version is found

### PR DESCRIPTION
**Pull request by ofek**
### Motivation

Follow-up to https://github.com/DataDog/datadog-agent/pull/3657

### CODE REVIEW

**Review by olivielpeau — APPROVED** [review]
LGTM.

please let the release manager @hush-hush know before merging 👍

**Review by hush-hush — APPROVED** [review]

**Pull request merged**
